### PR TITLE
Adding a simple wiki for distcp-ng events

### PR DESCRIPTION
**Pull request by chavdar**
- Added a simple wiki that documents the events sent by the Distcp-ng publisher
- Added code to CopyDataPublisher to use a more readable value if the origin/update timestamps are missing.

@ibuenros @pcadabam @ydai1124 can you review?

### CODE REVIEW

File: gobblin-docs/data-management/DistcpNgEvents.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    - `partition` - the partition (dataset-specific) that was copied
+    - `originTimestamp` - the timestamp of the dataset partition as generated in the origin (e.g. the database)
+    - `upstreamTimestamp` - the timestamp of the dataset partition as made available in the upstream system; this will be equal to `originTimestamp` if the data is read directly from the origin.
+
+- `DatasetPublishFailed` - a dataset failed to publish 
+    - `datasetUrn` - the URN of the dataset (dataset-specific) whose publish failed
+
+
+- `FilePublished` - sent when an individual file gets published
+    - `datasetUrn` - the URN of the dataset (dataset-specific) of which the part is part
```
Comment (ydai1124) [inline gobblin-docs/data-management/DistcpNgEvents.md]: This sentence for explaining `datasetUrn` seems incomplete.

**Comment by ydai1124** [comment]
Small comment. LGTM

**Comment by pcadabam** [comment]
LGTM other than that one comment

File: gobblin-docs/data-management/DistcpNgEvents.md (hunk 1 above)
Comment (chavdar) [inline gobblin-docs/data-management/DistcpNgEvents.md]: Fixed.

**Pull request merged**